### DISCUSSION TITLE
How long does it take for a new Release to be published in the Terraform Registry?

### DISCUSSION THREAD

**Discussion by justincandoit** [Snowflake-Labs/terraform-provider-snowflake#1664] [Q&A]
I am looking forward to some new features in the latest release and have been checking in for the past couple of days on the [terraform registry](https://registry.terraform.io/providers/Snowflake-Labs/snowflake/latest), but am still seeing 0.59 as the latest. 
Is there a timeline for when 0.60 will be available in the registry?

**Comment by culpgrant** [comment]
I just looked and 6.0 is up there now - I don't know how long it takes or what that process is.
